docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+    title: 'React Native with MOXY',
+    tagline: 'MOXY\'s boilerplate to create React Native applications',
+    url: 'https://react-native-with.moxy.tech',
+    baseUrl: '/',
+    favicon: 'img/favicon.ico',
+    organizationName: 'moxystudio',
+    projectName: 'react-native-with-moxy',
+    themeConfig: {
+        algolia: {
+            apiKey: '08e70f81e1b9b6e6c8c5bd149eaa9ff3',
```

Comment:
I belive we must apply for algolia since the website/domain/project is different. This means that the `apiKey` here won't work.

Reply:
Indeed, the configuration of the domain should be done in first place, and then apply to algolia to have an valid api key

Reply:
@AfonsoVReis, has this been addressed already?

Reply:
I applied for https://docsearch.algolia.com/docs/apply/ yesterday but im still waiting for the confirmation snippet, i can update this to be empty for now and submit a new PR when the confirmation arrives. What do you think?

Reply:
I guess so 😄 
Please create an issue here in GitHub and assign it to yourself so we don't forget to follow-up later.